pyro/poutine/block_messenger.py

```diff
@@ -23,6 +23,14 @@ class BlockMessenger(Messenger):
     Then any poutine outside of BlockMessenger(fn, hide=["a"])
     will not be applied to site "a" and will only see site "b":
 
+    >>> import pyro
+    >>> import pyro.distributions as dist
+    >>> from pyro.poutine.trace_messenger import TraceMessenger
```

Comment:
How is it that you don't need to import `BlockMessenger`? Does doctest automatically import the class/function being documented?

Reply:
Yes, that would be within the scope. Also any file imports will be in the docstring's global scope, and won't need to be imported.